test/integration/dashboard_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class DashboardTest < ActionDispatch::IntegrationTest
-  def setup
+  before(:all) do
```

Comment:
Why :all?  Wouldn't each be equivalent rspec style?  (or just plain `setup` for test unit)